website/src/pages/hackathons.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react'
+import Helmet from 'react-helmet'
+import Layout from '../components/Layout'
+import { eventLogger } from '../EventLogger'
+
+export default class Pricing extends React.Component<any, any> {
+    public componentDidMount(): void {
+        const script = document.createElement('script')
+        script.src = '//js.hsforms.net/forms/v2.js'
+        const hubspot = document.getElementById('hubspotTrialForm')
+        hubspot.appendChild(script)
+        script.addEventListener('load', () => {
+            ;(window as any).hbspt.forms.create({
+                portalId: '2762526',
+                formId: '7d6c55af-3de3-4e57-a5df-a0de341a4814',
+                target: '#hubspotTrialForm',
+                onFormSubmit: (e: Event) => {
+                    eventLogger.trackHackathonFormSubmitted()
+                },
+            })
+        })
+    }
+
+    public render(): JSX.Element | null {
+        return (
+            <Layout location={this.props.location}>
+                <div>
+                    <Helmet>
+                        <title>Sourcegraph - Set up Sourcegraph at a Hackathon</title>
+                        <meta name="twitter:title" content="Set up Sourcegraph at a Hackathon" />
+                        <meta property="og:title" content="Set up Sourcegraph at a Hackathon" />
+                    </Helmet>
+                    <div className="pricing-bg" />
+                    <div className="jumbotron text-right dark-7">
+                        <div className="row">
+                            <div className="sales py-0 d-flex flex-grow flex-column flex-lg-row align-items-center justify-content-around">
+                                <div>
+                                    <h1 className="text-center">Set up Sourcegraph during a hackathon</h1>
+                                    <p className="text-center measure">
+                                        Hackathons are the perfect time to set up code search and intelligence for your
```

Comment:
I'd just cut down on this section — feels a bit long for a subtitle (unless this is our standard)